README.md

```diff
@@ -57,41 +60,103 @@ If you are interested in becoming a member of the team check out the **[Pixelogi
 We found [this guide](https://www.digitalocean.com/community/tutorials/understanding-the-gopath) pretty helpful in understanding the Golang file structure and how it should be setup. We have tweaked this file structure to fit our Firebase Function setup.
 
 1. Make sure to [install Golang](https://golang.org/dl/) we are using v1.13
-2. Find your `GOPATH`. ALWAYS located in this path (`$HOME/go`) unless put otherwise
-   1. Example GOPATH: `Users/YourComputerName/go`
-   2. Example GOPATH/BIN:`Users/YourComputerName/go/bin`
-3. Add GOPATH env variable
+1. Find your `GOPATH`. ALWAYS located in this path (`$HOME/go`) unless put otherwise
+    1. Example GOPATH: `Users/YourComputerName/go`
+    1. Example GOPATH/BIN:`Users/YourComputerName/go/bin`
+1. Add GOPATH env variable
```

Comment:
[nit]

Looks like the numbering here is incorrect

Reply:
I always number with all 1. 's it is fully supported by markdown (which will number accordingly). And with this practice it allows you to reorder the list very easily.

Reply:
If you don't like this formatting I will happily set them back. I found it easier to deal with and thought I would suggest it to the team.

Reply:
Oh snap I actually did not know this about markdown! Good to know for sure. No worries at all this is great. Thanks for teaching me something new :)